include/roaring/roaring.h

```diff
@@ -94,8 +95,49 @@ void roaring_bitmap_printf_describe(const roaring_bitmap_t *r);
 
 /**
  * Creates a new bitmap from a list of uint32_t integers
+ *
+ * This function is deprecated, use `roaring_bitmap_from` instead, which
+ * doesn't require the number of elements to be passed in.
+ *
+ * @see roaring_bitmap_from
```

Comment:
Nit: the `@see` form of documentation is not used so far in the codebase, consider using natural language to conform.